services/121-service/src/user/user.service.ts

```diff
@@ -197,8 +186,7 @@ export class UserService {
       id: user.id,
       email: user.email,
       token: this.generateJWT(user),
-      role: user.role,
-      status: user.status,
+      roles: user.roles,
```

Comment:
(See elsewhere; the only thing to return would be the `token` (which would contain the whole `user`).